Library/Homebrew/test/os/dependency_collector_spec.rb

```diff
@@ -0,0 +1 @@
+require "test/os/linux/dependency_collector" if OS.linux?
```

Comment:
This can just be `dependency_collector_spec.rb` without the `if` or the `os/linux` file.